src/test/java/org/apache/sling/commons/mime/internal/MimeTypeServiceImplTest.java

```diff
@@ -43,6 +44,18 @@ public class MimeTypeServiceImplTest extends TestCase {
 
     private static final String TEXT_PLAIN = "text/plain";
 
+    private static final String EPS = "eps";
+
+    private static final Map<String, String> epsMimeTypeExt = new HashMap<>();
+
+    static {
+        epsMimeTypeExt.put("application/eps", EPS);
```

Comment:
I think it's enough to test two different mime types, right? Otherwise we overspecify the behaviour. And for two tests you can use two assertions instead of a map and a loop in the test.

Reply:
There are 4 extensions that have been added.  I like to test them all